Ash.FrontEnd/src/NavBar/style.less

```diff
@@ -1,25 +1,41 @@
 // Navigation Bar styles
 @import '~variables.less';
 
-// TODO: Flesh out styles
 .ash-navbar {
   height: 50px;
-  border-bottom: solid 1px @app-border-color;
-  padding-left: 20%;
+  width: calc(90% - @left-gutter-width);
+  margin-left: @left-gutter-width;
+  padding-left: 1%;
   display: flex;
   align-items: flex-end;
 
-  a {
-    margin-right: 10px;
-    padding: 5px;
-    border-left: solid 1px @app-border-color;
-    border-top: solid 1px @app-border-color;
-    border-right: solid 1px @app-border-color;
+  .ash-logo {
+    position: fixed;
+    top: 20px;
+    left: 20px;
+    height: 150px;
+    width: 150px;
+    padding: 20px;
+    white-space: pre;
+    color: #000;
```

Comment:
Do we want pure white/black to be variables or just leave as is?

Reply:
Eh just leave as is.  This is a placeholder anyways.